ext/ripper/depend

```diff
@@ -12,7 +12,7 @@ ripper.o: ripper.c
 
 .y.c:
 	$(ECHO) compiling compiler $<
-	$(Q) $(BISON) -t -v -o$@ -h$*.h - parse.y < $<
+	$(Q) $(BISON) -t -v -o$@ -h$*.h - ripper.y < $<
```

Comment:
```suggestion
	$(Q) $(BISON) -t -v -o$@ -h$*.h - $< < $<
```

Reply:
Or `$(top_srcdir)/parse.y`

Reply:
`$(srcdir)/tools/preproc.rb` processes `parse.y` to generate `ripper.y` then I think `ripper.y` (`$<`) is better than `parse.y`.